components/atoms/input/index.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import classNames from 'classnames'
+
+import styles from './styles.css'
+
+const Input = ({
+  className,
+  type,
+  checked,
+  disabled,
+  id,
+  name,
+  multiline,
+  placeholder,
+  required,
+  value,
+  handleChange,
+}) => {
+  const props = {
+    className: classNames(styles.input, className, { [styles.disabled]: disabled }),
+    disabled,
+    id,
+    name,
+    placeholder,
+    required,
+    value,
+    onChange: handleChange,
+  }
+  return multiline ? <textarea {...props} /> : <input {...props} type={type} checked={checked} />
+}
+
+Input.propTypes = {
+  className: PropTypes.string,
+  type: PropTypes.string,
+  checked: PropTypes.bool,
+  disabled: PropTypes.bool,
+  id: PropTypes.string,
+  name: PropTypes.string,
+  placeholder: PropTypes.string,
+  multiline: PropTypes.bool,
+  required: PropTypes.bool,
+  value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  handleChange: PropTypes.func.isRequired,
```

Comment:
I think that `handleChange` should be optional and the default should be `() => {}`